labgraph/lsl_node/__init__.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
Would you please create and move LSL node code to following folder https://github.com/facebookresearch/labgraph/devices/protocol/lsl/ ?